prow/config/tide_test.go

```diff
@@ -113,6 +113,7 @@ func TestMergeMethod(t *testing.T) {
 		MergeType: map[string]github.PullRequestMergeType{
 			"kubernetes/kops":             github.MergeRebase,
 			"kubernetes/charts":           github.MergeSquash,
+			"helm/charts":                 github.MergeSquash,
```

Comment:
Don't think this should change

Reply:
this has to change for the PR to pass, separately this test should be in config/ something since it is kube specific.

Reply:
Really? This test looks like a test for the `MergeMethod()` function off of this test config ... a little confused how this is passing actually with this change but nothing changing on line 138

Reply:
It appears bazel CI runs are checking the real status of the merge method. So, when I simply changed the "kubernetes/charts" to "helm/charts" things failed. That's why both are here. I expect we can remove the "kubernetes/charts" after the move.

Reply:
ew :|

Reply:
yeah I don't know what is up here, I'd cricle back to reconsider what's going on with this test after this PR, but patching it to pass like this seems like the right thing to do within the scope of this PR